### PR TITLE
Add JAVA_OPTIONS so proxy settings get picked by jenkins-plugin-cli

### PR DESCRIPTION
**Pull request by wazery**
### What this PR does / why we need it
Adds a necessary environment variable to the README, to make proxy settings work for `jenkins-plugin-cli` to work

### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

### Checklist

- [x] [DCO](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] CHANGELOG.md was updated

### CODE REVIEW

File: charts/jenkins/README.md [hunk 1]
```diff
@@ -853,6 +853,8 @@ controller:
       value: "http://192.168.64.1:3128"
     - name: no_proxy
       value: ""
+    - name: _JAVA_OPTIONS
```
Comment (timja) [inline charts/jenkins/README.md:856]: why `_` ?

Comment (wazery) [reply charts/jenkins/README.md]: Removed it

File: charts/jenkins/CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The change log until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 3.9.4
+
+Add _JAVA_OPTIONS to the README so proxy settings get picked by jenkins-plugin-cli
```
Comment (timja) [inline charts/jenkins/CHANGELOG.md:17]: please correct the variable name here and in PR title

Comment (wazery) [reply charts/jenkins/CHANGELOG.md]: Done

**Review by timja — APPROVED** [review]

**Pull request merged**